### ISSUE TITLE
Add WCAG fail for secondary buttons to accessibility statement

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-design-system#4918]
## What

We've learned about a [WCAG fail of 2.4.7 Focus Visible on secondary buttons](https://github.com/alphagov/govuk-frontend/issues/4469#issuecomment-3397323886) that was previously undiscovered.
We need to adjust the accessibility statement to include it.

## Why

To keep the accessibility statement legally compliant and make other people aware of the issue.

## Who needs to work on this

Accessibility specialist, content designer

## Who needs to review this

Accessibility specialist?, content designer?

## Done when

- [ ] Addition to accessibility statement is drafted
- [ ] Changes to accessibility statement are published
- [ ] (Maybe) changes have been communicated to the community
